.swift-format

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Just out of curiosity: how did you decide on all these settings?

Reply:
I’m using the same ones we were already using for daily-ios. =)